metals/src/main/scala/scala/meta/internal/metals/doctor/ScalaProblem.scala

```diff
@@ -42,6 +42,12 @@ case class DeprecatedScalaVersion(version: String) extends ScalaProblem {
       s"please upgrade to Scala ${ScalaVersions.recommendedVersion(version)}."
 }
 
+case class DeprecatedRemovedScalaVersion(version: String) extends ScalaProblem {
+  override def message: String =
+    s"Scala $version is no longer being bugfixed, " +
+      s"please upgrade to Scala ${ScalaVersions.recommendedVersion(version)}."
+}
```

Comment:
I mean, IDE should work, but no recent bugfixes will be present. I doubt that this could potentially break. My hope is that this will work infinetely.

Reply:
Hmm, what about `IDE support for Scala $version is working, but it's no longer being bugfixed and improved. To get the best experience, consider upgrading to Scala ...` 

What I'm trying to achieve is to reduce ambiguity of `Scala $version is no longer being bugfixed` as I personally read this as referring to compiler version, not IDE support